fenix/app/src/main/res/values/strings.xml

```diff
@@ -290,8 +290,9 @@
         The first parameter is the name of the app defined in app_name (for example: Fenix) -->
     <string name="juno_onboarding_default_browser_title" tools:ignore="UnusedResources">Make %s your go-to browser</string>
     <!-- Description for set firefox as default browser screen.
-        The first parameter is the string with key "juno_onboarding_default_browser_description_link_text", that says "privacy notice" in en-* -->
-    <string name="juno_onboarding_default_browser_description" tools:ignore="UnusedResources">Firefox puts people over profits and defends your privacy by blocking cross-site trackers.\n\nLearn more in our %s.</string>
+        The first parameter is the Firefox brand name.
```

Comment:
nit regarding consistency: For this kind of scenarios the description usually is
> The first parameter is the name of the app defined in app_name (for example: Fenix)

Reply:
hmm, I think it's not the `app_name` but the brand name in this scenario, which should say "Firefox" even on beta and nightly, right?

App name is for the string on line 291.